Menu/Tab/index.jsx

```diff
@@ -82,6 +84,16 @@ export default React.createClass({
     }
   },
 
+  componentWillUnmount () {
+    window.removeEventListener('resize', this.resizeListener)
+  },
+
+  resizeListener () {
+    window.requestAnimationFrame
+      ? window.requestAnimationFrame(() => update(this))
```

Comment:
Yeah, I distinctly remember writing something like this before, but I don't remember where... Maybe somewhere in the instalments component? 😄